### PR TITLE
PCEngines fw update

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

**Comment by TomaszAIR** [comment]
@PLangowski please target this PR to develop not main

File: meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh [hunk 1]
```diff
@@ -566,6 +566,63 @@ board_config() {
...
+          DASHARO_REL_NAME="pcengines_apu2"
+          DASHARO_REL_VER="0.9.0"
+          HAVE_EC="false"
+          NEED_EC_RESET="false"
+          BIOS_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER/${DASHARO_REL_NAME}_v${DASHARO_REL_VER}.rom"
+          BIOS_HASH_LINK_COMM="${BIOS_LINK_COMM}.sha256"
+          BIOS_SIGN_LINK_COMM="${BIOS_LINK_COMM}.sha256.sig"
+          PROGRAMMER_BIOS="internal:boardmismatch=force"
+          NEED_SMMSTORE_MIGRATION="false"
+          NEED_BOOTSPLASH_MIGRATION="false"
```
Comment (macpijan) [inline meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh:582]: here as well

File: meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh [hunk 2]
```diff
@@ -566,6 +566,63 @@ board_config() {
...
+        "APU2")
+          DASHARO_REL_NAME="pcengines_apu2"
+          DASHARO_REL_VER="0.9.0"
+          HAVE_EC="false"
+          NEED_EC_RESET="false"
+          BIOS_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER/${DASHARO_REL_NAME}_v${DASHARO_REL_VER}.rom"
+          BIOS_HASH_LINK_COMM="${BIOS_LINK_COMM}.sha256"
+          BIOS_SIGN_LINK_COMM="${BIOS_LINK_COMM}.sha256.sig"
+          PROGRAMMER_BIOS="internal:boardmismatch=force"
+          NEED_SMMSTORE_MIGRATION="false"
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh]: Done cfbb21e54071747883fd83eef3f27cc573a8e0d1

File: meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh (hunk 1 above)
Comment (PLangowski) [reply meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh]: Done cfbb21e54071747883fd83eef3f27cc573a8e0d1

**Review by macpijan — APPROVED** [review]
LGTM, any more comments @mkopec ?

**Comment by mkopec** [comment]
Looks good in general, but the board paths are different atm, so we'll need to either change them here or update the dasharo build and upload scripts to ise this path instead.

Right now all binaries are placed in directory `pcengines_apu2/v0.9.0` (the boards are treated as variants of apu2)

**Comment by macpijan** [comment]
So @PLangowski please adjust the script to match the paths as suggested by @mkopec

**Comment by PLangowski** [comment]
@macpijan done a6f8ce96a4a3eb711a1fdb151d24ef082f0eae58

**Review by macpijan — APPROVED** [review]

**Pull request merged**